endringslogg/src/main/kotlin/no/bekk/env/Env.kt

```diff
@@ -0,0 +1,11 @@
+package no.bekk.env
+
+val DB_USERNAME: String = System.getenv("DB_USERNAME") ?: System.getenv("USER")
+val DB_PASSWORD = System.getenv("DB_PASSWORD") ?: ""
+val DB_HOST = System.getenv("DB_HOST") ?: "localhost"
+val DB_DATABASE = System.getenv("DB_DATABASE") ?: "endringslogg"
+val DB_PORT = System.getenv("DB_PORT") ?: "5432"
+val SANITY_PROJECT_ID: String = System.getenv("SANITY_PROJECT_ID") ?: "li581mqu"
```

Comment:
Assumption is made that NAV will only have one project. Alternative could be to pass this through frontend component.